### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team, please review my code for cart page
- structured the page.
- added styling to the page
- linked the page to other pages.

Live preview: https://stylevilla-cart.netlify.app/cart.html

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+.price #strike {
+  text-decoration: line-through;
+  font-size: 1.6rem;
+  color: #9a9a9a;
+}
+
+.discount {
+  color: #9a9a9a;
```
Comment (tushargoyal812) [inline css/cart.css]: use color variables

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+.product-card .total {
+  font-weight: 700;
+  border-bottom: 1px solid black;
+  border-top: 1px solid black;
+  padding: 10px 0;
+  margin-top: 1.2rem;
+}
+
+.price-details #price-detail {
+  float: right;
```
Comment (tushargoyal812) [inline css/cart.css:103]: suggestion: using float is not  good practice,use flex properties for "justify-content:flex-end or flex-start"

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+.total #price-detail {
+  float: right;
+}
+
+.product-card .note {
+  padding: 10px 0;
```
Comment (tushargoyal812) [inline css/cart.css:111]: use rems

**Review by tushargoyal812 — COMMENTED** [review]
only some minor changes,looks good to merge

**Review by chanchal-panpaliya — APPROVED** [review]
Looks Good To Merge

File: css/cart.css (hunk 1 above)
Comment (sahiba0915) [reply css/cart.css]: done

**Pull request merged**